build.gradle

```diff
@@ -186,53 +186,9 @@ allprojects {
   project.version = scmVersion.version
 }
 
-def maintainerclean = tasks.register('maintainerclean') {
-  group = "Workflow"
-  description = "clean all submodules"
-  def tb = taskBuilder('clean')
-  codeModules.each { dependsOn tb(it) }
-  dockerModules.each { dependsOn tb(it) }
-  //depender codeModules, taskBuilder('clean')
-  //depender dockerModules, taskBuilder('clean')
-}
-
-def dockerclean = tasks.register('dockerclean') {
-  group = "Workflow"
-  description = "clean docker images"
-  dependsOn ':app:dockerComposeDown'
-  dependsOn ':app:dockerPrune'
-}
-
-def dockermaintainerclean = tasks.register('dockermaintainerclean') {
-  group = "Workflow"
-  description = "clean docker images"
-  dependsOn ':app:dockerPruneVolume'
-  mustRunAfter dockerclean
-}
-
-def devloop = tasks.register('devloop') {
```

Comment:
This and subsequent tasks assume a single Docker setup, which is no longer true.